monitoring/v3/src/main/java/CreateCustomMetric.java

```diff
@@ -40,12 +39,12 @@
 import java.util.TimeZone;
 
 
-
-
 /**
- * Class to demonstrate creating a custom metric with Cloud Monitoring. This class provides a few
- * functions that create a custom GAUGE metric, writes a timeseries value to it, then reads that
- * metric's value back within the last 5 minutes to see the value written.
+ * Class to demonstrate creating a custom metric with Cloud Monitoring.
+ * <p/>
```

Comment:
I think the end p tags `</p>` are optional in HTML5. I don't think I've seen many javadocs that use them.